importer/importer/models.py

```diff
@@ -260,11 +269,13 @@ def get_item_images(self, item_id):
         # check whether the folder contains the number of items it should
         actual_item_count = len(os.listdir(destination_path))
         if self.collection_data[item_id]["size"] != actual_item_count:
-            self.logger.error("Should have {0} images for item {1} but instead have {2} images".format(
-                self.collection_data[item_id]["size"],
-                item_id,
-                actual_item_count
-            ))
+            self.logger.error("Should have %(expected_count)d images for item %(item_id)s but "
+                              "instead have %(actual_count)d images",
+                              {'expected_count': self.collection_data[item_id]["size"],
+                               'item_id': item_id,
+                               'actual_count': actual_item_count
+                               }
```

Comment:
Previously I'd ask about the linespace / indentation style consistency here but now I'm inclined to defer that to “just run Black”